### PR TITLE
Practices: Adds Lean Experiments

### PR DESCRIPTION
**Pull request by joemoore**

### CODE REVIEW

File: content/practices/lean-experiments/index.md [hunk 1]
```diff
@@ -0,0 +1,173 @@
+---
+title: "Lean Experiments"
+linkTitle: "Lean Experiments"
+description: "A way to have your team create experiments to run so that you can validate or invalidate risky assumptions that may lead to product failure"
+tags: ["Framing", "Inception"]
```
Comment (fanibahar) [inline content/practices/lean-experiments/index.md:5]: Delivery

File: content/practices/lean-experiments/index.md [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+If there are many assumptions to validate, consider following up with a hypothesis/experiment tracker. This can help the team visualize what hypotheses should be tested in what order, and their associated experiments.
+
+## Related Practices
+[Assumptions](/practices/assumptions)
+
+## Variations
+None at the moment. 
+
+### Preceding
+None at the moment.
```
Comment (fanibahar) [inline content/practices/lean-experiments/index.md:160]: Assumptions

File: content/practices/lean-experiments/index.md [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+1. **Generate experiments**
+   
+   Distribute a sheet of paper to each team member and instruct them to:
+   
+   Copy the format of the Hypothesis & Experiment template
+   Write the leap-of-faith assumption as a falsifiable hypothesis
+   If there are more than 4 participants, consider breaking into teams to generate experiments
+   
+   {{< callout >}}
+   Tip: Print out your templates, email/share them, or create and share the digital workspace before the workshop.
```
Comment (fanibahar) [inline content/practices/lean-experiments/index.md:106]: Copy the lean experiment template from this miro board: https://miro.com/app/board/o9J_lGThxtc=/

File: content/practices/lean-experiments/index.md [hunk 4]
```diff
@@ -0,0 +1,173 @@
...
+   {{< callout >}}
+   Tip: Prompt everyone to take 5 minutes individually to think of an experiment to test the hypothesis and associated measurable outcomes.
+   {{< /callout >}}
+
+1. **Share experiments**
+   
+   When time is up, have each team member (one by one) stick their sheet up on the wall and share the details of their experiment for group discussion.
+   
+   {{< callout >}}
+   Tip: Iteration on experiments from team feedback should be encouraged.
```
Comment (fanibahar) [inline content/practices/lean-experiments/index.md:118]: Give everyone 5 minutes to share their experiments

File: content/practices/lean-experiments/index.md [hunk 5]
```diff
@@ -0,0 +1,173 @@
...
+   For any of the outcomes, record the results of the experiment and planned next steps. A good resource to leverage is [The Learning Card](https://assets.strategyzer.com/assets/resources/the-learning-card.pdf) from Strategyzer.
+  
+   {{< callout >}}
+   Tip: It’s good practice to set a super fail condition on your experiment. This is a failure so bad that it could indicate the experiment itself has an issue. We should believe something to be true in order to test it, so an epic failure should be unlikely. 
+   {{< /callout >}}
+
+1. **Plan next steps & action items**
+   
+   Brainstorm and then assign actions for team members to take in order to execute the experiment.
+
```
Comment (fanibahar) [inline content/practices/lean-experiments/index.md:146]: 1. **Create experiment tracker 
Keep the not selected experiments in this session into your backlog. Regularly review the experiment backlog to keep executing the experiment that is relevant to validate your product direction

**Review by fanibahar — COMMENTED** [review]
1. Adding tags "delivery" because lean experiment is still needed during product development
2. Adding miro link that includes lean experiment template
3. Adding facilitator tip to set timebox during share experiment share
4. Additional step to create experiment tracker for not selected experiment
5. Preceding practices should be assumptions exercise

File: content/practices/lean-experiments/index.md (hunk 1 above)
Comment (joemoore) [reply content/practices/lean-experiments/index.md]: Updated

File: content/practices/lean-experiments/index.md (hunk 3 above)
Comment (joemoore) [reply content/practices/lean-experiments/index.md]: Updated

File: content/practices/lean-experiments/index.md (hunk 4 above)
Comment (joemoore) [reply content/practices/lean-experiments/index.md:123]: Updated

File: content/practices/lean-experiments/index.md (hunk 5 above)
Comment (joemoore) [reply content/practices/lean-experiments/index.md:157]: Updated.

File: content/practices/lean-experiments/index.md (hunk 2 above)
Comment (joemoore) [reply content/practices/lean-experiments/index.md]: Added

**Comment by joemoore** [comment]
Note: This PR is related to issue #805

**Comment by joemoore** [comment]
Feedback incorporated. Ready for merge.

**Review by BrianMMcClain — APPROVED** [review]

**Pull request merged**